src/components/CollectionsAndPalettes/common/CollectionsToolbar/CollectionsToolbar.tsx

```diff
@@ -111,7 +187,19 @@ export default class CollectionsToolbar extends React.Component<Props, any> {
       <div className={styles.toolbarContainer}>
         <div className={styles.toolbarInfo}>
           <div className={styles.titleFlex}>
-            <div className={styles.title}>{this.props.title}</div>
+            {this.state.isRenameMode ? (
+              <input
+                ref={this.wrapperRefRename}
+                className={styles.renameInput}
```

Comment:
Maybe we can make this input not look like an input? If we match the font size, weight and color of the normal title display, and we use no border and a transparent background-color, we might be able to make this look like a "live edit".